### PR TITLE
Improvements for the SCM/CI Integration

### PR DESCRIPTION
**Pull request by dmarcoux**
Preview of the new note about multibuild flavors:

preview-note

### CODE REVIEW

File: xml/obs_scm_ci_workflow_integration.xml [hunk 1]
```diff
@@ -1124,7 +1131,7 @@ workflow:
         source_project: GNOME:Factory
         source_package: gnome-shell
         target_project: home:jane:playground
-   - configure_repositories:
+    - configure_repositories:
```
Comment (dmarcoux) [inline xml/obs_scm_ci_workflow_integration.xml:1134]: Thank you @perlpunk for reporting this :+1:

File: xml/obs_scm_ci_workflow_integration.xml [hunk 2]
```diff
@@ -778,20 +778,27 @@ workflow:
         </mediaobject>
       </figure>
 
-      <para>Moreover, if your package builds several multibuild flavours, the status will
+      <para>Moreover, if your package builds several multibuild flavors, the status will
```
Comment (eduardoj) [inline xml/obs_scm_ci_workflow_integration.xml:781]: Any reason to change from `flavours` to `flavor`? It seems that `flavor` is used is the preferred spelling in the United States, while `flavour` is used the rest of the English-speaking world.

**Review by eduardoj — APPROVED** [review]

File: xml/obs_scm_ci_workflow_integration.xml (hunk 2 above)
Comment (dmarcoux) [reply xml/obs_scm_ci_workflow_integration.xml:781]: In the SCM/CI integration, we used both. See below the line 783 with the link to https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.multibuild.html. On this page, we always use `flavor`, so this explains why I decided to go with `flavor`.

Comment (eduardoj) [reply xml/obs_scm_ci_workflow_integration.xml:781]: I see. Thanks!

**Pull request merged**